nodejs/eks/examples/migrate-nodegroups/steps/step1/index.ts

```diff
@@ -0,0 +1,99 @@
+import * as awsx from "@pulumi/awsx";
```

Comment:
This looks like a lot of duplicated code from the example. Any easy way to dedup?